### PR TITLE
upgraded to latest hyperkube

### PR DESCRIPTION
**Pull request by cheld**
Upgraded to latest Kubernetes (1.3-alpha) and did some clean up

@batikanu can you review?

---
This change is [Reviewable](https://reviewable.io/reviews/kubernetes/dashboard/770)

### CODE REVIEW

**Comment by cheld** [comment]
Switched back to stable version

File: build/hyperkube.sh [hunk 1]
```diff
@@ -40,14 +36,16 @@ docker run \
...
     --privileged=true \
     -d \
-    gcr.io/google_containers/hyperkube:v${K8S_VERSION} \
-    /hyperkube kubelet --containerized --hostname-override="127.0.0.1" \
-        --address="0.0.0.0" --api-servers=http://localhost:${PORT} \
-        --config=/etc/kubernetes/manifests
-
-docker run -d --net=host --privileged gcr.io/google_containers/hyperkube:v${K8S_VERSION} \
-    /hyperkube proxy --master=http://127.0.0.1:${PORT} --v=2
+    gcr.io/google_containers/hyperkube:${K8S_VERSION} \
```
Comment (luxas) [inline build/hyperkube.sh]: `hyperkube` is deprecated. Use `hyperkube-amd64` instead.

**Comment by batikanu** [comment]
Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/770)*

**Comment by batikanu** [comment]
LGTM

<details><summary><i>Previously, cheld (Christoph Held) wrote…</i></summary>
> Switched back to stable version 
</details>


---

Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/770#-:-KIlFqH8EtrMsmL1kAo5:676834233)*

**Comment by batikanu** [comment]
LGTM

**Comment by cheld** [comment]
*[build/hyperkube.sh, line 39 \[r1\]](https://reviewable.io:443/reviews/kubernetes/dashboard/770#-KIbbQ_THQdLr91uLlaJ-r1-39:-KJ-JvKwms6-jK6jvto5:-2137399250) ([raw file](https://github.com/kubernetes/dashboard/blob/a2bfd81e0780b3c46a526defc4990ef22f02b230/build/hyperkube.sh#L39)):*
<details><summary><i>Previously, luxas (Lucas Käldström) wrote…</i></summary>
> `hyperkube` is deprecated. Use `hyperkube-amd64` instead.
</details>
Thanks

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/kubernetes/dashboard/770)*

**Comment by bryk** [comment]
Please merge :)

**Comment by cheld** [comment]
ok

**Pull request merged**